src/prognostic_equations/edmfx_entr_detr.jl

```diff
@@ -68,14 +70,15 @@ function entrainment(
         # non-dimensional pi-groups
         Π₁ = (ᶜz - z_sfc) * (ᶜbuoyʲ - ᶜbuoy⁰) / ((ᶜwʲ - ᶜw⁰)^2 + eps(FT)) / 100
         Π₂ = max(ᶜtke⁰, 0) / ((ᶜwʲ - ᶜw⁰)^2 + eps(FT)) / 2
-        Π₃ = sqrt(ᶜaʲ)
+        Π₃ = sqrt(max(ᶜaʲ, 0))
```

Comment:
FYI you don't need this as if ᶜaʲ <= 0 we return 0 for entrainment above. But it's fine to be extra safe.